trust-quorum/gfss/src/shamir.rs

```diff
@@ -134,7 +134,7 @@ impl std::fmt::Debug for Share {
 
 pub struct SecretShares {
     pub threshold: ValidThreshold,
-    pub shares: Secret<Vec<Share>>,
+    pub shares: SecretBox<Vec<Share>>,
```

Comment:
This should also use `into_boxed_slice` and return a `SecretBox<[Share]>`.

This almost certainly should have been a boxed slice to begin with and not a `Vec`. So thanks for fixing it :)